tests/src/Appender/RollingFileAppenderTest.cs

```diff
@@ -103,7 +104,8 @@ private static void ResetAndDeleteTestFiles()
 			Utils.GetRepository().Shutdown();
 			((Repository.Hierarchy.Hierarchy)Utils.GetRepository()).Clear();
 
-			DeleteTestFiles();
+			DeleteTestFiles(c_fileName);
```

Comment:
I would like to preserve the method with signature `private static void DeleteTestFiles()` and move these two invocations there.